app/src/main/java/ca/rmen/android/poetassistant/main/dictionaries/ResultListFragment.java

```diff
@@ -221,7 +222,7 @@ private void updateUi() {
         // we'll show a text to tell them to search.
         if (TextUtils.isEmpty(query)) {
             String emptySearch = getString(R.string.empty_list_without_query);
-            ImageSpan imageSpan = new ImageSpan(getActivity(), R.drawable.ic_action_search_dark);
```

Comment:
* ImageSpan would store the context and resource id, and later call `mContext.getDrawable(mResourceId);`
* mContext is an AppCompatActivity
* Context.getDrawable() does `getResources().getDrawable(id, getTheme());`
* AppCompatActivity used to override `getResources()`:
```
    @Override
    public Resources getResources() {
        if (mResources == null) {
            mResources = new TintResources(this, super.getResources());
        }
        return mResources;
    }
```

So previously, it would be a TintResources which would get the drawable. This would use support-library classes which manage vectors.

Now, AppCompatActivity no longer overrides getResources().  So it's the framework's getResources() which gets the drawable, and this only supports vectors from v21+.